File: 4. Remove all vowels from the string/code.java
import java.util.Scanner;

public class Main {
    public static String removeVowels(String s) {
        String vowels = "aeiouAEIOU";
        StringBuilder result = new StringBuilder();
        for (char c : s.toCharArray()) {
            if (vowels.indexOf(c) == -1) {
                result.append(c);
            }
        }
        return result.toString();
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a string: ");
        String inputStr = scanner.nextLine();
        System.out.println("String after removing vowels: " + removeVowels(inputStr));
    }
}

File: 4. Remove all vowels from the string/code.cpp
#include <iostream>
#include <string>
using namespace std;

string removeVowels(string s) {
    string vowels = "aeiouAEIOU";
    string result = "";
    for (char c : s) {
        if (vowels.find(c) == string::npos) {
            result += c;
        }
    }
    return result;
}

int main() {
    string input_str;
    cout << "Enter a string: ";
    getline(cin, input_str);
    cout << "String after removing vowels: " << removeVowels(input_str) << endl;
    return 0;
}
